elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -188,11 +189,37 @@ export class PeerPool extends EventEmitter {
 	private readonly _peerSelectForConnection: P2PPeerSelectionForConnectionFunction;
 	private readonly _sendPeerLimit: number;
 	private readonly _outboundShuffleIntervalId: NodeJS.Timer | undefined;
+	private readonly _inboundPeerConfig: PeerConfig;
+	private readonly _outboundPeerConfig: PeerConfig;
 
 	public constructor(peerPoolConfig: PeerPoolConfig) {
 		super();
 		this._peerMap = new Map();
 		this._peerPoolConfig = peerPoolConfig;
+		this._inboundPeerConfig = {
```

Comment:
why does `inbound` have no config for `banTime` and `wsMaxPayload` but outbound does

Reply:
`wsMaxPayload` is not necessary for inbound because it is set on the SocketCluster server (not on the socket). It seems that `banTime` is not used in any of the peer classes. I guess we can remove it.